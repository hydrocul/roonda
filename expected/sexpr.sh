echo Hello
echo Hello World
echo 'Hello   World!'
echo roonda_0ac56896a1a3f1427f602202defd62faaa1cc4c8.txt

#################################################
# roonda_0ac56896a1a3f1427f602202defd62faaa1cc4c8.txt:
#################################################
# 
# Hello Hello
# 
