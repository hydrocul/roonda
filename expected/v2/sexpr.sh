
#################################################
# attachedfile.txt:
#################################################
# 
# Hello Hello
# 

#################################################
# roonda_3772d206f212590b290e2779a784416fef72956e.pl:
#################################################
# print "Hello\n";
echo Hello
echo Hello World
echo 'Hello   World!'
cat $ROONDA_TMP_PATH/attachedfile.txt
perl $ROONDA_TMP_PATH/roonda_3772d206f212590b290e2779a784416fef72956e.pl
perl $ROONDA_TMP_PATH/roonda_3772d206f212590b290e2779a784416fef72956e.pl
