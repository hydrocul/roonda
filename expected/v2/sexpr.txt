Hello
Hello World
Hello   World!

Hello Hello

Hello
Hello
